modules/migration-enabling-cached-kubernetes-clients.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration-toolkit-for-containers/advanced-migration-options-mtc.adoc
+
+[id="migration-enabling-cached-kubernetes-clients_{context}"]
+= Enabling cached Kubernetes clients
+
+You can enable cached Kubernetes clients in the `MigrationController` custom resource (CR) for improved performance during migration. The greatest performance benefit is displayed when migrating between clusters in different regions or with significant network latency.
+
+[NOTE]
+====
+Delegated tasks, for example, Rsync backup for direct volume migration or Velero backup and restore, however, do not show improved performance with cached clients.
+====
+
+Cached clients require extra memory because the `MigrationController` CR caches all API resources required for interacting with `MigCluster` CRs. Requests that would normally go to the API server are directed to the cache instead. The cache watches the API server for updates.
```

Comment:
@eriknelson @djwhatle 
Can we be specific about how much extra memory is required to enable cached clients?